src/renderer/globals.scss

```diff
@@ -5,12 +5,20 @@
 $primary: #519fc2;
 $primary-invert: findColorInvert($primary);
 
+$purple: #32075a;
+$purple-invert2: findColorInvert($purple);
+
+$orange: #dd6300;
+$orange-invert: findColorInvert($orange);
+
 $colors: (
   "white": ($white, $black),
   "black": ($black, $white),
   "light": ($light, $light-invert),
   "dark": ($dark, $dark-invert),
   "primary": ($primary, $primary-invert),
+  "purple": ($purple, $purple-invert),
```

Comment:
Why purple? Could we keep our Design System in black mode with our blue? Or our blue a little darken?